vizro-core/docs/pages/user_guides/container.md

```diff
@@ -78,7 +92,7 @@ To add a [`Container`][vizro.models.Container] to your page, do the following:
         Vizro().build(dashboard).run()
         ```
 
-        1.  If you don't specify `layout`, it will default to `[[0], [1]]`, meaning the components will be arranged in rows.
+        1.  Note that the `Page.layout` argument is not specified here and will therefore default to `[[0], [1]]`, meaning the containers will be stacked in rows.
```

Comment:
```suggestion
        1.  Note that the `Page.layout` argument is not specified here and will therefore default to `[[0], [1]]`, meaning the containers will be stacked in rows.
         2. Stack the components inside this `Container` in columns.
```

Reply:
I just realised that this phrasing is basically the contrary of what flexbox does. If they are stacked below each other, they call it "stacked in one column", if they are stacked side by side, they call it "stacked in one row".

Maybe we use that instead?

Reply:
Yeah, that's much clearer I think 👍 To align with our new convention on the other page I'd suggest using the words horizontal and vertical as well as explaining non-ambiguously what it means like you do:
```suggestion
        1. Note that the `Page.layout` argument is not specified here and will therefore default to `[[0], [1]]`, meaning the containers will be **vertically stacked** down the page in one column.
        2. **Horizontally stack** the components side-by-side inside this `Container` in one row.
```